### PR TITLE
migrate to public uxd client

### PR DESCRIPTION
**Pull request by Orelsanpls**

### CODE REVIEW

File: hooks/useGovernanceUnderlyingTokenAccounts.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import BigNumber from 'bignumber.js';
 import { useCallback, useEffect, useState } from 'react';
 import { Connection, PublicKey } from '@solana/web3.js';
 import { BN } from '@project-serum/anchor';
-import { findATAAddrSync } from '@uxdprotocol/uxd-client';
+import { findATAAddrSync } from '@uxd-protocol/uxd-client';
```
Comment (sjillen) [inline hooks/useGovernanceUnderlyingTokenAccounts.ts]: this `findATAAdrSync` helper is present in utils/ataTools (or somewhere around there) I believe. We should not rely on our uxd client package to find this helper, but have it locally

File: tools/sdk/lifinity/withdrawFromPool.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { TOKEN_PROGRAM_ID } from '@solana/spl-token';
 import { SignerWalletAdapter } from '@solana/wallet-adapter-base';
 import { Connection, PublicKey } from '@solana/web3.js';
-import { findATAAddrSync } from '@uxdprotocol/uxd-client';
+import { findATAAddrSync } from '@uxd-protocol/uxd-client';
```
Comment (sjillen) [inline tools/sdk/lifinity/withdrawFromPool.ts]: same here (though it's probably my fault, default import probably fucked that up)

File: tools/sdk/quarryMine/instructions/createMiner.ts [hunk 3]
```diff
@@ -10,7 +10,7 @@ import {
   SystemProgram,
   TransactionInstruction,
 } from '@solana/web3.js';
-import { findATAAddrSync } from '@uxdprotocol/uxd-client';
+import { findATAAddrSync } from '@uxd-protocol/uxd-client';
```
Comment (sjillen) [inline tools/sdk/quarryMine/instructions/createMiner.ts]: same

File: tools/sdk/quarryMine/instructions/stakeTokens.ts [hunk 4]
```diff
@@ -7,7 +7,7 @@ import {
 import { AugmentedProvider } from '@saberhq/solana-contrib';
 import { TOKEN_PROGRAM_ID } from '@solana/spl-token';
 import { PublicKey, TransactionInstruction } from '@solana/web3.js';
-import { findATAAddrSync } from '@uxdprotocol/uxd-client';
+import { findATAAddrSync } from '@uxd-protocol/uxd-client';
```
Comment (sjillen) [inline tools/sdk/quarryMine/instructions/stakeTokens.ts]: same

File: tools/sdk/quarryMine/instructions/withdrawTokens.ts [hunk 5]
```diff
@@ -7,7 +7,7 @@ import {
 import { AugmentedProvider } from '@saberhq/solana-contrib';
 import { TOKEN_PROGRAM_ID } from '@solana/spl-token';
 import { PublicKey, TransactionInstruction } from '@solana/web3.js';
-import { findATAAddrSync } from '@uxdprotocol/uxd-client';
+import { findATAAddrSync } from '@uxd-protocol/uxd-client';
 import QuarryMineConfiguration, { SupportedMintName } from '../configuration';
```
Comment (sjillen) [inline tools/sdk/quarryMine/instructions/withdrawTokens.ts]: same

**Review by sjillen — CHANGES_REQUESTED** [review]
not your fault, but importing findATAAddrSync from uxd-client adds a dependency that we should not have

**Comment by Orelsanpls** [comment]
Good catch @ThomasProust  :)

**Review by sjillen — APPROVED** [review]
cheers!

File: yarn.lock [hunk 6]
```diff
@@ -2997,13 +3018,16 @@
...
+  version "1.0.0"
+  resolved "https://registry.yarnpkg.com/@uxd-protocol/uxd-client/-/uxd-client-1.0.0.tgz#93a41fc75724bef84bcf8f03c40fadf7b963ffad"
+  integrity sha512-wI2Zg0xhRzPPDGwKgJfvJdC1fI7x96IVdXWi53u7Mw8OZSRSCS7JjoKFiJ0K5QPE4eSYKXJdQh7ChKtviK1pnQ==
   dependencies:
-    "@blockworks-foundation/mango-client" "3.3.15"
-    camelcase "^5.3.1"
+    "@blockworks-foundation/mango-client" "3.4.4"
+    "@project-serum/anchor" "0.22.1"
+    "@solana/spl-token" "0.1.8"
+    "@solana/web3.js" "1.35.1"
```
Comment (0xCryptoSheik) [inline yarn.lock]: Opportunity to align some dependencies in `package.json` assuming you use them in your code (remove them from `package.json` if not used in your code & not required in peerDependencies), no need to bother with `resolutions`.

File: tools/sdk/uxdProtocol/uxdClient.ts [hunk 7]
```diff
@@ -6,7 +6,7 @@ import {
   findAddrSync,
```
Comment (0xCryptoSheik) [inline tools/sdk/uxdProtocol/uxdClient.ts]: do you have this util locally as well?

**Comment by 0xCryptoSheik** [comment]
+1 :)

File: yarn.lock (hunk 6 above)
Comment (sjillen) [reply yarn.lock]: we really need to get work done on the dependencies... seriously

**Pull request closed without merging**